### ISSUE TITLE
How to used in Ios 7?

### ISSUE THREAD

**Issue by zhangxu273** [nikki2pro/YSC_SpineSwiftTest#1]
Greetings,I see the code used "SKAction.unhide()" & "SKAction.hide()" , but It not support on Ios7. how can I do it?

**Comment by zhangxu273** [comment]
OK i try to solve it.
I create a class

```
class YSC_SpineUtils {
     static func customHideAction() -> SKAction{
        return SKAction.customActionWithDuration(0, actionBlock: { (node, time) -> Void in
            node.hidden = true;
        })
    }

     static func customUnhideAction() -> SKAction{
        return SKAction.customActionWithDuration(0, actionBlock: { (node, time) -> Void in
            node.hidden = false;
        })
    }
}
```

e.g.
before

```
 finalActionSequence = [SKAction.unhide(), SKAction.group(SRTActionGroup)]
```

after

```
        if #available(iOS 8.0, *) {
            finalActionSequence = [SKAction.unhide(), SKAction.group(SRTActionGroup)]
        } else {
            finalActionSequence = [YSC_SpineUtils.customUnhideAction(),SKAction.group(SRTActionGroup)]
        }
```

**Comment by nikki2pro** [comment]
Thank you for your interest in my work. I should update this project so that it works on the latest Swift version.
